### PR TITLE
add Arineo GmbH

### PR DESCRIPTION
**Pull request by Florian-Pflueger**
**Description**
Changes proposed in this pull request:
- added Arineo GmbH as adopter

### CODE REVIEW

File: content/adopters/adopters.yaml [hunk 1]
```diff
@@ -20,4 +20,8 @@ adopters:
     content: "Kyma enables us to quickly develop flexible and robust solutions to extend our clients cloud-based and on-premise enterprise applications."
   - websiteUrl: "https://www.arithnea.de/"
     logo: "./logos/arithnea.svg"
-    content: "Kyma supports us with tools to create a unique customer experience with SAP CX - one that is even more efficient, flexible, and at low risk."
\ No newline at end of file
+    content: "Kyma supports us with tools to create a unique customer experience with SAP CX - one that is even more efficient, flexible, and at low risk."
+  - websiteUrl: "https://www.arineo.com/"
+    url: "https://arineo.com/de/customer-experience/177-iot-with-sap"
+    logo: "./logos/arineo.svg"  
+    content: "Kyma solves our clients’ need for a flexible and reusable SAP Commerce extension mechanism. It allows them to offer an extraordinary CX."
```
Comment (kazydek) [inline content/adopters/adopters.yaml]:
```suggestion
    content: "Kyma solves our clients’ need for a flexible and reusable SAP Commerce extension mechanism. It allows them to deliver extraordinary customer experience."
```

Comment (magicmatatjahu) [reply content/adopters/adopters.yaml]: @Florian-Pflueger Can you apply changes? Then I will run jobs and merge. Thanks for contribution! :)

**Pull request merged**